.markdownlint.jsonc

```
@@ -219,6 +219,12 @@
         "replace": "$1plain",
         "searchScope": "text",
       },
+      {
+        "name": "gfm-blockquote",
+        "message": "Use the GFM syntax: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#notes_warnings_and_callouts",
+        "searchPattern": "/^ *> \\*\\*(Note|Warning):\\*\\*/gm",
```

```suggestion
        "searchPattern": "/^ *> \\*\\*(Note|Warning):\\*\\*(?! [[{`_*])/gm",
```

There's a Prettier bug that prevents these from being applied correctly.

---

Also need to check `Callout`

---

There is no `**Callout:**` in the repo, but callout in old syntax works.

```suggestion
        "searchPattern": "/^ *> \\*\\*(Note|Warning|Callout):\\*\\*(?! [[{`_*])/gm",
```